### ISSUE TITLE
Tab bar background disappears when reaching bottom when miniplayer is not enable 

### ISSUE THREAD

**Issue by leogiroux** [LeoNatan/LNPopupController#481]
No issue when player is showing : 
![File](https://user-images.githubusercontent.com/12122348/134490345-8c8f11bb-97a9-44f0-bd64-076fae825b3e.jpg)

No issue when the view scroll is not a the bottom : 
![File (1)](https://user-images.githubusercontent.com/12122348/134490351-414b0dde-7169-4a12-9e12-34de402ce260.jpg)

Issue occurred when scrolling at the top and beyond the safe area
![File (2)](https://user-images.githubusercontent.com/12122348/134490362-a740d21f-7c30-48cd-a44f-dfe8918d97e3.jpg)

This is my controllers inside a UITabBarController

```        
       self.viewControllers = [ homeVC, searchVC, downloadVC ].map {
            return UINavigationController(rootViewController: $0)
        }
```
and this how I update the appearence

```
  private func initAppearance() {
        print("initAppearance")
        
        let e = UITabBarAppearance()
        e.backgroundColor = ThemeManager.currentTheme().backgroundColor.withAlphaComponent(0.8)
        e.backgroundEffect = UIBlurEffect(style: .systemChromeMaterialLight)
        e.selectionIndicatorTintColor = ThemeManager.currentTheme().textColor
        tabBar.tintColor = ThemeManager.currentTheme().textColor
        tabBar.standardAppearance = e
        tabBar.setNeedsLayout()
    
        
        
        let appearance = LNPopupBarAppearance()
        appearance.titleTextAttributes = [.font: UIFont.rounded(ofSize: 16, weight: .semibold), .foregroundColor: ThemeManager.currentTheme().textColor]
        appearance.subtitleTextAttributes = [.font: UIFont.rounded(ofSize: 12, weight: .regular), .foregroundColor: ThemeManager.currentTheme().textColor]
        appearance.marqueeScrollEnabled = true
        appearance.backgroundColor = ThemeManager.currentTheme().backgroundColor.withAlphaComponent(0.8)
        appearance.backgroundEffect = UIBlurEffect(style: .systemChromeMaterialLight)
        
        let appearanceProxy = LNPopupBar.appearance()
        appearanceProxy.inheritsAppearanceFromDockingView = false
        appearanceProxy.progressViewStyle = .top
        appearanceProxy.progressView.tintColor = .white
        appearanceProxy.tintColor = ThemeManager.currentTheme().textColor
        appearanceProxy.standardAppearance = appearance

    }
```

I didn't had this issue before, just update to the last version.
Sorry for the duplicate ticket I got an issue with the computer.

**Comment by LeoNatan** [comment]
Can you please create a small project that reproduces this issue? I am unable to. Thanks

**Comment by leogiroux** [comment]
Oh I know the cause you have written inside your README file : 

Starting with iOS 15, scroll edge appearance is automatically disabled for toolbars and tab bars when a popup bar is presented, regardless of the scroll position of the content. Once the popup bar is dismissed, the scroll edge appearance is restored.

and the gif below is showing exactly my issue, do you know how to fix that ?

**Comment by LeoNatan** [comment]
But that’s only when the popup bar is presented. After you dismiss it, the system goes back to normal.

**Comment by leogiroux** [comment]
Ok I fix it, it was not from your library but from a new behaviour of IOS 15,

I add my fix code here if someone had the same issue than me :) thanks for your time 👍 
```
       if #available(iOS 15.0, *) {
            let appearance = UITabBarAppearance()
            appearance.configureWithOpaqueBackground()
            appearance.backgroundColor = ThemeManager.currentTheme().backgroundColor.withAlphaComponent(0.8)
            appearance.backgroundEffect = UIBlurEffect(style: .systemChromeMaterialLight)
            appearance.selectionIndicatorTintColor = ThemeManager.currentTheme().textColor
            tabBar.tintColor = ThemeManager.currentTheme().textColor
            self.tabBar.standardAppearance = appearance
            self.tabBar.scrollEdgeAppearance = tabBar.standardAppearance
        }
```

**Comment by LeoNatan** [comment]
👍
